### ISSUE TITLE
llama_get_logits_ith: invalid logits id 11, reason: no logits

### ISSUE THREAD

**Issue by zhaowanjin** [ollama/ollama#6501]
### What is the issue?

llama_get_logits_ith: invalid logits id 11, reason: no logits

### OS

Linux

### GPU

Nvidia

### CPU

Intel

**Comment by rick-github** [comment]
[Server logs](https://github.com/ollama/ollama/blob/main/docs/troubleshooting.md#how-to-troubleshoot-issues) will aid in debugging, but it would be helpful if you could provide context.

**Comment by zhaowanjin** [comment]
time=2024-08-25T23:29:34.047+08:00 level=INFO source=server.go:567 msg="llama runner started in 33.88 seconds"
[GIN] 2024/08/25 - 23:29:34 | 200 | 34.812771654s |       127.0.0.1 | POST     "/api/chat"
[1724599846] uhd_slice_image: multiple 9
[1724599846] uhd_slice_image: image_size: 1920 1440; source_image size: 518 392
[1724599846] uhd_slice_image: image_size: 1920 1440; best_grid: 3 3
[1724599846] uhd_slice_image: refine_image_size: 1554 1176; refine_size: 1554 1176
[1724599846] clip_image_preprocess: 518 392
[1724599846] clip_image_preprocess: 518 392
[1724599846] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_preprocess: 518 392
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 1 of 10 encoded in   127.88 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 2 of 10 encoded in    80.28 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 3 of 10 encoded in    76.35 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 4 of 10 encoded in    69.31 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 5 of 10 encoded in    68.69 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 6 of 10 encoded in    67.99 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 7 of 10 encoded in    68.63 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 8 of 10 encoded in    68.64 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 9 of 10 encoded in    68.93 ms
[1724599847] clip_image_build_graph: 518 392
[1724599847] encode_image_with_clip: step 10 of 10 encoded in    68.26 ms
[1724599847] encode_image_with_clip: all 10 segments encoded in   765.10 ms
[1724599847] encode_image_with_clip: load_image_size 1920 1440
[1724599847] encode_image_with_clip: image embedding created: 960 tokens
[1724599847] 
encode_image_with_clip: image encoded in   767.04 ms by CLIP (    0.80 ms per image patch)
llama_get_logits_ith: invalid logits id 9, reason: no logits
[GIN] 2024/08/25 - 23:30:48 | 200 |  2.571823672s |       127.0.0.1 | POST     "/api/chat"

![issues](https://github.com/user-attachments/assets/9eab1c9c-ad0f-4eb0-a089-2753517c8734)
